pymc3/distributions/continuous.py

```diff
@@ -488,32 +499,33 @@ class TruncatedNormal(Continuous):
         Mean.
     sd : float
         Standard deviation (sd > 0).
-    a : float (optional)
+    lower : float (optional)
```

Comment:
I agree that `lower` and `upper` are nicer names, but it _is_ different from `scipy.stats.truncnormal`.

Reply:
we already use `mu` and `sd` instead of `loc` and `scale`, so I guess two more doesnt hurt `¯\_(ツ)_/¯`

Also, since we are calling `sd` instead of `sigma`, pretty sure we lose the moral high ground to criticize naming in other packages...

Reply:
We should really fix the sd - sigma thing (my fault).